<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>InsertDialog</class>
    <widget class="QDialog" name="InsertDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <widget class="QWidget" name="">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>20</y>
                    <width>291</width>
                    <height>221</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QLabel" name="label">
                        <property name="text">
                            <string>Enter Your Matrix:</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QTextEdit" name="matrixEdit"/>
                </item>
                <item>
                    <widget class="QLabel" name="label_2">
                        <property name="text">
                            <string>Enter Your Vector:</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QLineEdit" name="vectorEdit"/>
                </item>
            </layout>
        </widget>
        <widget class="QWidget" name="">
            <property name="geometry">
                <rect>
                    <x>200</x>
                    <y>250</y>
                    <width>168</width>
                    <height>30</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <widget class="QPushButton" name="cancelButton">
                        <property name="text">
                            <string>Cancel</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="okButton">
                        <property name="text">
                            <string>Ok</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
